app/script_variant.py

```diff
@@ -168,6 +215,14 @@ def romanize_search_query(word):
         hiragana_word = jautils.normalize(word)
         romanized_words.append(jautils.hiragana_to_romaji(hiragana_word))
 
-    romanized_word = unidecode(word)
-    romanized_words.append(romanized_word.strip())
+    # if the name is a Chinese name and the chinese_romanize can produce
+    # a different result, append the result to the romanzied_words with
+    # unidecode results together
+    unidecode_romanize_word = unidecode(word).strip()
+    if is_chinese_person_name(word):
```

Comment:
It looks waste to split Chinese names twice (in is_chinese_person_name and romanize_chinese_person_name). How about removing this check, because romanize_chinese_person_name already has a logic when it fails to split?